docs/logs/README.md

```diff
@@ -0,0 +1,105 @@
+# OpenTelemetry .NET Logs
+
+## Best Practices
+
+The following tutorials have demonstrated the best practices for logging with
+OpenTelemetry .NET:
+
+* [Getting Started - Console Application](./getting-started-console/README.md)
+* [Getting Started - ASP.NET Core
+  Application](./getting-started-aspnetcore/README.md)
+* [Logging with Complex Objects](./complex-objects/README.md)
+
+## Structured Logging
+
+:heavy_check_mark: You should use structured logging.
+
+* Structured logging is more efficient than unstructured logging.
+  * Filtering and redaction can happen on invidual key-value pairs instead of
+    the entire log message.
+  * Storage and indexing are more efficient.
+* Structured logging makes it easier to manage and consume logs.
+
+:stop_sign: You should avoid string interpolation.
+
+> [!WARNING]
+> The following code has bad performance due to [string
+  interpolation](https://learn.microsoft.com/dotnet/csharp/tutorials/string-interpolation):
+
+```csharp
+var food = "tomato";
+var price = 2.99;
+
+logger.LogInformation($"Hello from {food} {price}.");
+```
+
+Refer to the [logging performance
+benchmark](../../test/Benchmarks/Logs/LogBenchmarks.cs) for more details.
+
+## Package Version
+
+:heavy_check_mark: You should always use the
+[`ILogger`](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger)
+interface from the latest stable version of
+[Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging/)
+package, regardless of the .NET runtime version being used:
+
+* If you're using the latest stable version of [OpenTelemetry .NET
+  SDK](../../src/OpenTelemetry/README.md), you don't have to worry about the
+  version of `Microsoft.Extensions.Logging` package because it is already taken
+  care of for you via [package dependency](../../Directory.Packages.props).
+* Starting from version `3.1.0`, the .NET runtime team is holding a high bar for
+  backward compatibility on `Microsoft.Extensions.Logging` even during major
+  version bumps, so compatibility is not a concern here.
```

Comment:
I think "high bar" might not be clear to everyone. Maybe something like...

* Starting from version `3.1.0` the .NET runtime team has committed to preserve backwards compatibility on the `Microsoft.Extensions.Logging` packages. There will be no breaking changes (within reason) across major version bumps.

Reply:
I don't think we can put such strong statement https://github.com/open-telemetry/opentelemetry-dotnet/pull/4920#discussion_r1346923479.